Sources/Fuzzilli/Lifting/ScriptWriter.swift

```diff
@@ -60,7 +60,7 @@ struct ScriptWriter {
     
     /// Emit one line of code.
     mutating func emit<S: StringProtocol>(_ line: S) {
-        assert(!line.contains("\n"))
+        //assert(!line.contains("\n")) Corpus contains multiline strings
```

Comment:
What exactly is the problem here?

Reply:
The corpus I've been using has strings in it containing newlines, which was triggering this assert, IIRC

Reply:
Ah you mean string literals? But is that even valid JS syntax?

Reply:
To be honest, that's what I assumed was the cause, but don't have a test case to verify. I'll track one down